GrassChoppers/Modules/Home/HomeLogicController.swift

```diff
@@ -25,11 +26,21 @@ class HomeLogicController: HomeLogicControlling {
         self.dataManager.delegate = self
     }
     
+    func viewDidLoad() {
+        
+        dataManager.requestInitialState()
```

Comment:
We just got the `viewDidLoad` message (from the ViewController), we need to request the initial data of the screen from the data manager (since all data lives in the data manager).